src/vos/tests/vts_ilog.c

```diff
@@ -400,6 +400,49 @@ do_update(daos_handle_t loh, daos_epoch_t epoch,
 	return 0;
 }
 
+struct version_cache {
+	uint32_t	vc_ver[2];
+	int		vc_idx;
+};
+
+static void
+version_cache_init(struct version_cache *vcache, uint32_t initial_version)
+{
+	memset(vcache, 0, sizeof(*vcache));
+	vcache->vc_ver[1] = initial_version;
+}
+
+static bool
+version_cache_fetch_helper(struct version_cache *vcache, daos_handle_t loh,
+			   bool expect_change)
+{
+	vcache->vc_ver[vcache->vc_idx] = ilog_version_get(loh);
+	if (expect_change) {
+		if (vcache->vc_ver[0] == vcache->vc_ver[1]) {
+			print_message("version unexpectedly matches: %d\n",
+				      vcache->vc_ver[0]);
+			return false;
+		}
```

Comment:
It can be handled by following check, so redundant.

Reply:
done